rxpm/src/main/kotlin/me/dmdev/rxpm/State.kt

```diff
@@ -9,7 +9,7 @@ import me.dmdev.rxpm.util.*
 
 /**
  * Reactive property for the [view's][PmView] state.
```

Comment:
Revert please. As in other cases there is no need to change docs. Consumer and observable still the main way. And accept is convenience. 
If you think we need to show info about accept method in docs, let's discuss it.

Reply:
Sure, missed that!